pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -89,27 +90,21 @@ type windows struct {
 }
 
 // New returns a new Windows instance constructed from the given WindowsVM
-// TODO: WINC-429 will replace WindowsVM with a MachineSet
-func New(vm types.WindowsVM, workerIgnitionEndpoint string, signer ssh.Signer) (Windows, error) {
-	// Validate the WindowsVM
-	if err := validate(vm); err != nil {
-		return nil, errors.Wrap(err, "error validating Windows VM")
-	}
-
+func New(ipAddress, instanceID, workerIgnitionEndpoint string, signer ssh.Signer) (Windows, error) {
```

Comment:
We already checked it within the Reconcile method, so we should not see any empty strings for it.

Reply:
So there is a dependency on how `New` is being called and it is possible it could be call outside of the call path given it is a free standing object. But I won't hold you to it. We just need to be aware that there could be a potential for future bugs based on how the `windows` object is used.